funnel/models/proposal.py

```diff
@@ -108,7 +109,10 @@ class Proposal(BaseScopedIdNameMixin, CoordinatesMixin, db.Model):
     slides = db.Column(db.Unicode(250), nullable=True)
     preview_video = db.Column(db.Unicode(250), default=u'', nullable=True)
     links = db.Column(db.Text, default=u'', nullable=True)
-    status = db.Column(db.Integer, default=PROPOSALSTATUS.SUBMITTED, nullable=False)
+
+    _status = db.Column('status', db.Integer, StateManager.check_constraint('status', PROPOSALSTATUS),
+        default=PROPOSALSTATUS.SUBMITTED, nullable=False)
+    status = StateManager('_status', PROPOSALSTATUS, doc="The proposal's status")
```

Comment:
Change this to `state`. We'll keep the db column name `status` for now -- or throw in a migration.